README.md

```diff
@@ -29,7 +29,7 @@ The only build currently supported is Master Quest (Debug), but other versions a
 
 It builds the following ROM:
 
-* oot-gc-eu-mq-dbg.z64 `md5: f0b7f35375f9cc8ca1b2d59d78e35405`
+* oot_uncompressed-gc-eu-mq-dbg.z64 `md5: f0b7f35375f9cc8ca1b2d59d78e35405`
```

Comment:
It seems a bit strange to me that final output ROM, `oot-gc-eu-mq-dbg.z64`, does not match the debug ROM. What about a `COMPRESS` makefile variable, where if COMPRESS=0 then the uncompressed ROM is directly copied to the output ROM? It would be set to 1 for retail versions and 0 for debug, but modders could override it if they want their mod to be compressed.

Reply:
Personally I think running `make compress` and getting an uncompressed rom on some versions but not on others is bad.
I would like to hear more people's opinions about this

Reply:
Im not sure the best approach to solve this, but I do think itd be a bit weird if `oot-gc-eu-mq-dbg.z64` is not the name for the definitive matching debug rom

Reply:
I tbh am not sure where I stand on this. 
Theres two lenses to look at this through: OoT historical documentation/decomp mindset, and a general use (modding) codebase.

Logically it makes sense to label the uncompressed rom as uncompressed, and I do recognize that, but from an OoT "historian" viewpoint, it makes sense for the "not standard" format to have the "non standard" name.

Reply:
is `COMPARE` at the top supposed to be `COMPRESS`?
and  "oot-$(VERSION)-uncompressed.z64 is the "compressed ROM"", I assume you mean is the uncompressed rom?

So if I am following, for gc-eu-mq-dbg, we are going to get both a "uncompressed rom" and an "output rom", and both are going to be the exact same in that specific case?

Reply:
Yes, apologies for the sloppiness.

Yeah, if you do not want compression, the output rom is an exact copy of the uncompressed rom

Reply:
ok, I think this is the best sounding compromise so far

Reply:
> Logically it makes sense to label the uncompressed rom as uncompressed

I don't agree. Generating the rom in it's decompressed form should be considered standard output for compilation, so it shouldn't need "uncompressed" tacked on to the end. On the other hand, compressed roms should be labeled as being compressed, since it makes it more clear that a transformation on the original data has occurred. 

Arguably, we may also want to specify the type of compression that has been applied to the rom, since not all versions use the same compression algorithm.

Reply:
I keep flip flopping back and forth here, but I guess I do agree that "uncompressed" should probably be unlabeled, and compressed which has data altered should be labeled as such.
I do not agree on including the compression algorithm in the name tho, I dont find it necessary

Reply:
My way of seeing it is since the final result of the build process will produce a rom that matches the contents of the cartridge then that one should be favored and be unlabelled.
Another alternative would be to label both: `oot-compressed.z64` and `oot-uncompressed.z64`

Reply:
Yeah I mentioned above that the 2 main ways of looking at it above is "oot rom historian" or "result from a generic build pipeline". And while I understand the historian viewpoint, I think just naming them after how it objectively is (the "game" and the "compressed game"), is much more consistent and better long term.

we can try to document somewhere which ones are originally not compressed and compressed

Reply:
I do recognize the compromise option of labeling both, but do think mzx has a point here that labeling something as "uncompressed" seems a little redundant. It is just the output of make which builds the game. It get compressed after the fact to create a new variation of that original output. So I think "uncompressed" being unlabeled does make alot of sense here